css-canvas-medical/index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+<div class="top-container">
+  <div class="header">
+    <h1>Canvas</h1>
+  </div>
+  <div class="right-container">
+    <p class="top-right"> Product Implementation About Us Blog </p>
+    <button class="top-button"> Contact Us</button>
+  </div>
+</div>
+<hr>
+<div class="first-section">
+  <div class="inside">
+    <div class="emr-head">
+      <h1>The EMR Reimagined for<br>
+        Primary Care</h1>
+    </div>
+    <div class="text-under-head">
+      <p>Can't remember the last time you talked with a patient without focusing<br>
+        on the EMR? Canvas lets you rediscover joy in the work you do. Capture<br>
+        meaningful data, identify gaps in care, and move seamlessly between<br>
+        task within the perform. You stay present with patients while your<br>
+        whole care team stays up-to-speed. Be a better doctor with, canvas,<br>
+        designed for primary care provided with the help of primary Care<br>
+        providers.
+      </p>
+    </div>
+    <button class="middle-button">Get Startred</button>
+
+    <div class="overlay"></div>
+  </div>
+</div>
+<div class="back-ground">
+  <div class="second-section">
+    <div class="second-head">
+      <h2>Clinicians Prefer Canvas</h2> <br>
+    </div>
+    <div class="second-text">
+      <p>Physician-owned primary care practices across the country have<br>
+        already made Canvas the fabric of their care.</p>
+    </div>
+  </div>
+
+  <div class="two-columns">
+    <div class="column-one">
+      <p>Cascade Family Practice, Portland OR<br>
+        Temescal Creed Medicine, Oakland CA<br>
+        San Juan Healthcare, Friday Harbor WA
+      </p>
+    </div>
+    <div class="column-two">
+      <p> Ward Medical Clinic, Bonifay FL<br>
+        Caribou Medical, Towanda PA<br>
+        Bay Tree Family Care, Red Bay AL
+      </p>
+    </div>
+  </div>
+</div>
+
+<div class="last-content-row">
```

Comment:
be careful with names like `last-content-row` when there are more rows to come. I do stuff like this often, but make sure you go back and update names for semantic clarity!